test/NuGet.Clients.Tests/NuGet.PackageManagement.UI.Test/Models/Package/PackageModelFactoryTests.cs

```diff
@@ -66,5 +67,123 @@ public void Create_WithValidMetadata_ReturnsPackageModel()
             Assert.NotNull(result);
             Assert.Equal("TestPackage", result.Identity.Id);
         }
+
+        [Fact]
+        public void Create_RecommendedPackageSearchMetadata_AllTab_ReturnsRecommendedPackageModel()
```

Comment:
I recommend renaming other tests to match a pattern similar to this as well:

```suggestion
        public void Create_RecommendedPackageOnAllTab_ReturnsRecommendedPackageModel()
```